File: swagger/models/add_client_response.py
from .base_model import BaseModel

from .client import Client

class AddClientResponse(BaseModel):
	"""
	no description available

	Attributes:

		client
		Contains information about the client.
	"""

	input_map = {
		'Client': ('client', Client)
		}

	output_map = {
		'client': 'Client',
		}

File: swagger/Models/AddClientResponse.php
<?php

namespace App\Mindbody\Models;

/**
 * no description available
 */
class AddClientResponse extends BaseModel
{
	/**
	 * Contains information about the client.
	 * 
	 * @var Client 
	 */
	public $client;

	protected function getInputMap()
	{
		return [
			'Client' => ['client', Client::class, null],
			];
	}

	protected function getOutputMap()
	{
		return [
			'client' => ['Client', Client::class, null],
			];
	}
}
